.gitlab-ci.yml

```diff
@@ -37,3 +37,41 @@ test_cuda:
     - nvidia-smi
     - ctest --output-on-failure -E "^SeedingValidation/CompareWithActsSeedingTests.*"
 
+
+build_sycl:
+  tags: [docker]
```

Comment:
this can be removed. It was used to mimic what the shared GitLab runners were configured like, but that recently changed.